### ISSUE TITLE
JSON support - insert into issue

### ISSUE THREAD

**Issue by nirmaayan** [scylladb/scylla#3631]
Scylla version (or git commit hash): 2.3
OS (RHEL/CentOS/Ubuntu/AWS AMI): ami-a1c2c7de

When trying to do insert into we get - `SyntaxException: line 1:18 missing '(' at '<missing '`

Testing this issue in Cassandra and it works.

To reproduce:

#### Create table
```
CREATE TABLE nba.jtest (
    a text PRIMARY KEY,
    b timestamp
```

#### Insert into table

```
INSERT INTO jtest JSON '{  "a" : "Sprint",      "b" : "2011-02-03 04:05+0000" }';
```

#### Workaround 

```
INSERT INTO jtest (a, b) JSON '{  "a" : "Sprint" , "b" : "2011-02-03 04:05+0000" }'
```

**Comment by psarna** [comment]
The fix is already queued for merging here: https://groups.google.com/forum/#!topic/scylladb-dev/1D-2lhBhn6E

**Comment by nirmaayan** [comment]
@psarna FYI

**Comment by psarna** [comment]
Fixed with f66aace685c83fa81eaf2138e6be348cd1d1c765

**Comment by pdziepak** [comment]
Backport for 2.3 queued in next-2.3.
